app/routers/ai_chat.py

```
@@ -0,0 +1,192 @@
+            # Rough estimate for OpenAI
+            estimated_tokens = len(message.split()) * 1.3  # Rough token estimation
+            estimated_cost = (estimated_tokens / 1000) * 0.0006  # Output cost
+
+            return {
+                "provider": "openai",
+                "estimated_cost": round(estimated_cost, 6),
+                "estimated_tokens": int(estimated_tokens),
+                "complexity": complexity.value if provider == "auto" else "unknown",
+            }
+
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)